File: python/80_boys_and_girls_253A.py
n, m = map(int, input().split())
major, minor = ("B", "G") if n >= m else ("G", "B")
if m > n:
    major, minor = minor, major
    n, m = m, n
result = []
while n > 0 and m > 0:
    result.append(major)
    result.append(minor)
    n -= 1
    m -= 1
result.extend(major * n)
result.extend(minor * m)
print("".join(result))

File: c/80_boys_and_girls_253A.c
#include <stdio.h>
int main()
{
    int n, m;
    if (scanf("%d %d", &n, &m) != 2)
        return 0;
    char major = 'B', minor = 'G';
    if (m > n)
    {
        major = 'G';
        minor = 'B';
        int tmp = m;
        m = n;
        n = tmp;
    }
    while (n > 0 && m > 0)
    {
        putchar(major);
        putchar(minor);
        n--;
        m--;
    }
    while (n-- > 0)
        putchar(major);
    while (m-- > 0)
        putchar(minor);
    putchar('\n');
    return 0;
}
